From d7d5d75c17637ac68bc7b9ec38e39fbe915152af Mon Sep 17 00:00:00 2001
From: Jonas Eschle <jonas.eschle@cern.ch>
Date: Thu, 17 Feb 2022 13:47:49 +0100
Subject: [PATCH] Upgrade sklearn criterion mse to squared_error

mse is deprecated and will be removed: https://scikit-learn.org/stable/modules/generated/sklearn.tree.DecisionTreeRegressor.html

`squared_error` is equivalent.
---
 hep_ml/gradientboosting.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hep_ml/gradientboosting.py b/hep_ml/gradientboosting.py
index ebdae19..a36f8ed 100644
--- a/hep_ml/gradientboosting.py
+++ b/hep_ml/gradientboosting.py
@@ -118,7 +118,7 @@ def fit(self, X, y, sample_weight=None):
         for stage in range(self.n_estimators):
             # tree creation
             tree = SklearnClusteringTree(
-                criterion='mse',
+                criterion='squared_error',
                 splitter=self.splitter,
                 max_depth=self.max_depth,
                 min_samples_split=self.min_samples_split,
